bitnami/drupal/values.yaml

```diff
@@ -705,6 +705,91 @@ metrics:
   podAnnotations:
     prometheus.io/scrape: "true"
     prometheus.io/port: "9117"
+  ## Drupal exporter service parameters
+  ##
+  service:
+    ## @param metrics.service.type Drupal exporter service type
+    ##
+    type: ClusterIP
+    ## @param metrics.service.port Drupal exporter service port
+    ##
+    port: 9117
```

Comment:
```suggestion
    ports: 
      http-metrics: 9117
```

Reply:
@fmulero Do I need to rename "http-metrics" by "metrics" in those files to be consistent with other charts ?

Reply:
That could be great

Reply:
Done